std_msgs/Header header
int64 nAgentID
sensor_msgs/Image sImageMsg
---
int64 ack
